pkg/controller/deployable/deployable_controller.go

```diff
@@ -52,14 +54,127 @@ func Add(mgr manager.Manager) error {
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 	erecorder, _ := utils.NewEventRecorder(mgr.GetConfig(), mgr.GetScheme())
+	authClient := kubernetes.NewForConfigOrDie(mgr.GetConfig())
 
 	return &ReconcileDeployable{
 		Client:        mgr.GetClient(),
 		scheme:        mgr.GetScheme(),
+		authClient:    authClient,
 		eventRecorder: erecorder,
 	}
 }
 
+type placementruleMapper struct {
+	client.Client
+}
+
+func (mapper *placementruleMapper) Map(obj handler.MapObject) []reconcile.Request {
+	if klog.V(utils.QuiteLogLel) {
+		fnName := utils.GetFnName()
+		klog.Infof("Entering: %v()", fnName)
+		defer klog.Infof("Exiting: %v()", fnName)
+	}
+
+	cname := obj.Meta.GetName()
+	klog.V(10).Info("In placement Mapper:", cname)
+
+	var requests []reconcile.Request
+
+	dplList := &appv1alpha1.DeployableList{}
+	listopts := &client.ListOptions{}
+	err := mapper.List(context.TODO(), listopts, dplList)
+	if err != nil {
+		klog.Error("Failed to list deployables for placementrule mapper with error:", err)
+		return requests
+	}
+
+	for _, dpl := range dplList.Items {
+		if dpl.Spec.Placement == nil || dpl.Spec.Placement.PlacementRef == nil || dpl.Spec.Placement.PlacementRef.Name != obj.Meta.GetName() {
+			continue
+		}
+
+		annotations := dpl.GetAnnotations()
+		if annotations == nil || annotations[appv1alpha1.AnnotationManagedCluster] == "" {
+			continue
+		}
+
+		if annotations[appv1alpha1.AnnotationManagedCluster] != (types.NamespacedName{}).String() {
```

Comment:
Should it be 
```
if annotations[appv1alpha1.AnnotationManagedCluster] == (types.NamespacedName{}).String()  {
    continue
}
```

Reply:
this one is correct.  `!= "\"` means it is a deployable for managed cluster. we shall skip (continue)

Reply:
checked with Kuan, the comment is wrong  (types.NamespacedName{}).String() is "\", meaning the hub cluster.